hoot-cmd/src/main/cpp/hoot/cmd/main.cpp

```diff
@@ -77,7 +77,7 @@ int main(int argc, char *argv[])
     {
       return c->run(argv, argc);
     }
-    catch (std::exception& e)
+    catch (const std::exception& e)
```

Comment:
There are quite a few of these where catching an exception by `const` reference is the way to go.